File: verve patch/base/Applications/CSymphony_0_1/csSamples/CodeTypeDeclaration.cs
using System;
using System.CodeDom;
using System.CodeDom.Compiler;
using System.Collections;
using CSymphony.Lib;

namespace CSymphony.CSy
{
	public class CSyUnit : GraphUnit
	{
		public override void Build()
		{
			ns("MyNameSpace");

			declClass("TestType", "BaseType");

			declClass(TypePublic|TypeSealed, "TestdType2","BaseType", "BaseType2");

			declInterface("aa", "bb");

		}
	}
}

File: verve patch/base/Applications/CSymphony_0_1/vbSamples/CodeTypeDeclaration.vb
Option Strict Off
Option Explicit On

Imports System
Imports System.CodeDom
Imports System.CodeDom.Compiler
Imports System.Collections
Imports CSymphony.Lib

Namespace CSymphony.CSy
	Public Class CSyUnit
		 Inherits GraphUnit
		Overrides Public Sub Build()
			ns("MyNameSpace")

			declClass("TestType", "BaseType")

			declClass(TypePublic OR TypeSealed, "TestdType2","BaseType", "BaseType2")

			declInterface("aa", "bb")
		End Sub
	End Class
End Namespace
			